docs/cluster-agent/GETTING_STARTED.md

```diff
@@ -0,0 +1,225 @@
+
+If you are using the secret:
+
+```yaml
+          - name: DD_CLUSTER_AGENT_AUTH_TOKEN
+            valueFrom:
+              secretKeyRef:
+                name: datadog-auth-token
+                key: token
+
+```
+ 
+Or otherwise:
+ 
+```yaml
+          - name: DD_CLUSTER_AGENT_AUTH_TOKEN
+            value: "<32_CHARACTERS_LONG_TOKEN>"
+```
+
+**Note**: his needs to be set in the manifest of the cluster agent **AND** the node agent.
+
+**Step 4** - Once the secret is created, create the DCA along with its service.
+Don't forget to add your `<DATADOG_API_KEY>` in the manifest of the DCA. Both manifests can be found in the [manifest/cluster-agent directory](https://github.com/DataDog/datadog-agent/tree/master/Dockerfiles/manifests)
+Run: 
+
+`kubectl apply -f Dockerfiles/manifests/cluster-agent/datadog-cluster-agent_service.yaml`
+
+and
+
+`kubectl apply -f Dockerfiles/manifests/cluster-agent/cluster-agent.yaml`
+
+**Step 5** - At this point, you should be seeing:
+
+```
+-> kubectl get deploy
+NAME                    DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+datadog-cluster-agent   1         1         1            1           1d
+
+-> kubectl get secret
+NAME                   TYPE                                  DATA      AGE
+datadog-auth-token     Opaque                                1         1d
+
+-> kubectl get pods | grep cluster
+datadog-cluster-agent-8568545574-x9tc9   1/1       Running   0          2h
+
+-> kubectl get service
+NAME                  TYPE           CLUSTER-IP       EXTERNAL-IP        PORT(S)          AGE
+dca                   ClusterIP      10.100.202.234   <none>             5005/TCP         1d
+```
+
+**Step 6** - Configure your agent to communicate with the DCA.
+First, create the RBAC for your agents. They limit the agents' access to the kubelet API. For this create a dedicated:
+
+- Service Account
+- Cluster Role
+- Cluster Role Binding
+
+Those can be found in the [Datadog Agent Rbac](../../Dockerfiles/manifests/cluster-agent/rbac/rbac-agent.yaml).
+
+**Step 7** - Run: `kubectl apply -f Dockerfiles/manifests/cluster-agent/rbac-agent.yaml` from the datadog-agent directory.
+
+To do so, add the following environment variables to the agent's manifest:
+
+```yaml
+          - name: DD_CLUSTER_AGENT
+            value: 'true'
+          - name: DD_CLUSTER_AGENT_AUTH_TOKEN
+            valueFrom:
+              secretKeyRef:
+                name: datadog-auth-token
+                key: token 
+#            value: "<32_CHARACTERS_LONG_TOKEN>" # If you are not using the secret, just set the string.                
+          - name: DD_KUBERNETES_METADATA_TAG_UPDATE_FREQ # Optional
+            value: '15'
+```
+
+**Step 8** - Create the Daemonsets for your agents:
+
+`kubectl apply -f Dockerfiles/manifests/agent.yaml` 
+
+You should be seeing:
+
+```
+-> kubectl get pods | grep agent
```

Comment:
Why not selecting by labels ? It's fancy and more efficient
`kubectl get po -l key:value`